CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+[Unreleased]: https://github.com/SumoLogic/sumologic-otel-collector/compare/v0.50.0-sumo-0...main
+
+## [v0.50.0-sumo-0]
+
+### Released 2022-04-29
+
+Aside from upstream changes, this release only contains a performance fix to metrics batching in the Sumo Logic exporter.
+The performance improvement is very substantial, so we recommend upgrading to this version immediately after `0.49.0-sumo-0`.
+
 ### Added
 
 - feat: add rawk8seventsreceiver [#547][#547]
```

Comment:
Not strictly in this PR 🙂 but I think this should be removed from the changelog as long as the `rawk8sreceiver` is not included in the build.